### PR TITLE
Feature/jstricker/shell tutorial

### PR DESCRIPTION
**Pull request by johnstrickermayo**
The tutorial link is in the /documentation/rest-template-realizaiton.md page

if you want to test this out before merging:
* click the existing button in rest-template-realization.md to clone the repo
* in the shell console, switch to this branch: git checkout feature/jstricker/shell-tutorial
* run the tutorial: teachme cloud-shell-tutorial.md

### CODE REVIEW

File: simple-rest/cloud-shell-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# REST API with Big Table
+
+<walkthrough-tutorial-duration duration="10">
+</walkthrough-tutorial-duration>
+
+## Step 1: Edit the configuration file
+Edit the CONFIG file (currently open in the editor) to include:
+* account ID
```
Comment (clarkemily) [inline simple-rest/cloud-shell-tutorial.md]: should we specify more clearly that this is a billing account ID?

Comment (johnstrickermayo) [reply simple-rest/cloud-shell-tutorial.md]: good call out, yup, I'll push that up

File: simple-rest/cloud-shell-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Step 4: Create project infrastructure
+Run the `infrastructure.sh` script:
+```bash
+./infrastructure.sh
+```
+This script will: 
+* enable the APIs needed for Compute Engine and Big Table
+* create a Deployment Manager manifest file name `config-[your project id].yaml`
+* use the created manifest file to create a VM, firewall, and Big Table instance
```
Comment (clarkemily) [inline simple-rest/cloud-shell-tutorial.md]: this step also creates the CI/CD Build Trigger

Comment (johnstrickermayo) [reply simple-rest/cloud-shell-tutorial.md]: 👍

File: documentation/rest-template-realization.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+[![Open walkthrough in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/mkwalter0552/iac-templates.git&cloudshell_working_dir=simple-rest&cloudshell_open_in_editor=CONFIG%tutorial=cloud-shell-tutorial.md)
```
Comment (clarkemily) [inline documentation/rest-template-realization.md]: this opens a blank tab on mine and @morterer 's machines

File: simple-rest/cloud-shell-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+---
+[click here to report an issue with this tutorial](https://github.com/mkwalter0552/iac-templates/issues/new?title=[step%204%20problem]&labels=rest%20api%20shell%20tutorial,user%20feedback)
+
+<walkthrough-footnote>
+Note: google says in the fine print that enabling APIs can take up to 20 minutes, but we have yet to experience this script taking more then a couple of minutes.
+<walkthrough-footnote>
+
+## Step 5: Wire up your code
+Finally, you can then wire up your code to read and write to the Big Table instance. All you will need is the instance ID that is printed out as part of the `infrastructure.sh` script that you ran in the last step. How you wire up Big Table depends on your implementation
```
Comment (clarkemily) [inline simple-rest/cloud-shell-tutorial.md]: the user will need Project Id and Instance ID, although the IDs should be the same with our implementation.

File: documentation/rest-template-realization.md (hunk 3 above)
Comment (clarkemily) [reply documentation/rest-template-realization.md]: manually running teachme cloud-shell-tutorial after cloning and entering the directory myself works well.

**Comment by morterer** [comment]
This looks great, John!  I had to manually check out the code in Cloud Shell, but after that I was able to work thought the tutorial, and got the infrastructure up and running with no problems!!

**Pull request merged**